docs/tutorials/gke.md

```diff
@@ -0,0 +1,211 @@
+
+```console
+$ gcloud dns managed-zones create "external-dns-test-gcp-zalan-do" \
+    --dns-name "external-dns-test.gcp.zalan.do." \
+    --description "Automatically managed zone by kubernetes.io/external-dns"
+```
+
+Make a note of the nameservers that were assigned to your new zone.
+
+```console
+$ gcloud dns record-sets list \
+    --zone "external-dns-test-gcp-zalan-do" \
+    --name "external-dns-test.gcp.zalan.do." \
+    --type NS
+NAME                             TYPE  TTL    DATA
+external-dns-test.gcp.zalan.do.  NS    21600  ns-cloud-e1.googledomains.com.,ns-cloud-e2.googledomains.com.,ns-cloud-e3.googledomains.com.,ns-cloud-e4.googledomains.com.
+```
+
+In this case it's `ns-cloud-{e1-e4}.googledomains.com.` but your's could slightly differ, e.g. `{a1-a4}`, `{b1-b4}` etc.
+
+Connect your `kubectl` client to the cluster you just created.
+
+```console
+gcloud container clusters get-credentials "external-dns"
+```
+
+Apply the following manifest file to deploy `external-dns`.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        image: gcr.io/zalando-docker/external-dns:476c3efddff941ac1b168895c2ba1d5693adc8cc
+        args:
+        - --in-cluster
+        - --zone=external-dns-test-gcp-zalan-do
+        - --source=service
+        - --source=ingress
+        - --dns-provider=google
+        - --google-project=zalando-external-dns-test
+        - --dry-run=false
+```
+
+Create the following sample application to test that `external-dns` works.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: nginx.external-dns-test.gcp.zalan.do.
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 80
+  selector:
+    app: nginx
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        name: nginx
+        ports:
+        - containerPort: 80
+```
+
+After roughly two minutes check that a corresponding DNS record for your service was created.
+
+```console
+$ gcloud dns record-sets list \
+    --zone "external-dns-test-gcp-zalan-do" \
+    --name "nginx.external-dns-test.gcp.zalan.do." \
+    --type A
+NAME                                   TYPE  TTL  DATA
+nginx.external-dns-test.gcp.zalan.do.  A     300  104.155.60.49
+```
+
+Let's check that we can resolve this DNS name. We'll ask the nameservers assigned to your zone first.
+
+```console
+$ dig +short @ns-cloud-e1.googledomains.com. nginx.external-dns-test.gcp.zalan.do.
+104.155.60.49
+```
+
+If you hooked up your DNS zone with it's parent zone correctly you can use `curl` to access your site.
```

Comment:
> If you hooked up your DNS zone with it's parent zone correctly

Rather than questioning this at this point I would just briefly describe what is the correct configuration in the beginning.